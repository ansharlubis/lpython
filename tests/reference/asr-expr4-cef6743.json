{
    "basename": "asr-expr4-cef6743",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/expr4.py",
    "infile_hash": "5cba7a5d589f54fc31463e48903d5b46604fb64e3e64ba215339047c",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-expr4-cef6743.stdout",
    "stdout_hash": "449b4f7d84a50475cc57ad34d8d0934449807ec82439b82d48c67b33",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}