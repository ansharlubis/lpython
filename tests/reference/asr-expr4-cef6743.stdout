([1m[35mTranslationUnit[39m[0m 
   ([33mSymbolTable[39m 
      1 
      {
         test_del: 
            ([1m[35mSubroutine[39m[0m 
               ([33mSymbolTable[39m 
                  2 
                  {
                     a: 
                        ([1m[35mVariable[39m[0m 
                           2 
                           a [1m[32m
                           Local[39m[0m () () [1m[32m
                           Default[39m[0m 
                           ([1m[35mInteger[39m[0m 4 []) [1m[32m
                           Source[39m[0m [1m[32m
                           Public[39m[0m [1m[32m
                           Required[39m[0m .false.), 
                     b: 
                        ([1m[35mVariable[39m[0m 
                           2 
                           b [1m[32m
                           Local[39m[0m () () [1m[32m
                           Default[39m[0m 
                           ([1m[35mInteger[39m[0m 4 []) [1m[32m
                           Source[39m[0m [1m[32m
                           Public[39m[0m [1m[32m
                           Required[39m[0m .false.)
                  }) 
               test_del [] [
               ([1m[35m=[39m[0m 
                  ([1m[35mVar[39m[0m 2 [33ma[39m) ([1m[35mIntegerConstant[39m[0m [36m4[39m 
                  ([1m[35mInteger[39m[0m 4 [])) ()) 
               ([1m[35m=[39m[0m 
                  ([1m[35mVar[39m[0m 2 [33mb[39m) ([1m[35mIntegerConstant[39m[0m [36m20[39m 
                  ([1m[35mInteger[39m[0m 4 [])) ()) 
               ([1m[35mExplicitDeallocate[39m[0m [2 [33ma[39m 2 [33mb[39m])] [1m[32m
               Source[39m[0m [1m[32m
               Public[39m[0m [1m[32m
               Implementation[39m[0m () .false. .false.)
      }) 
   [])
(TranslationUnit (SymbolTable 1 {main_program: (Program (SymbolTable 3 {}) main_program [] []), test_del: (Subroutine (SymbolTable 2 {a: (Variable 2 a Local () () Default (Integer 4 []) Source Public Required .false.), b: (Variable 2 b Local () () Default (Integer 4 []) Source Public Required .false.)}) test_del [] [(= (Var 2 a) (IntegerConstant 4 (Integer 4 [])) ()) (= (Var 2 b) (IntegerConstant 20 (Integer 4 [])) ()) (ExplicitDeallocate [2 a 2 b])] Source Public Implementation () .false. .false.)}) [])
